src/getHumanAge.test.js

```diff
@@ -4,7 +4,38 @@ describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
   test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+    expect(getHumanAge).toBeInstanceOf(Function);
+  });
+
+  test('Should be an array', () => {
+    expect(getHumanAge()).toBeInstanceOf(Array);
```

Comment:
The test description 'Should be an array' might be misleading. If `getHumanAge` is supposed to return an array, the test should specify the expected content of the array for clarity.